libs/gltfio/src/FFilamentAsset.h

```diff
@@ -255,9 +258,10 @@ struct FFilamentAsset : public FilamentAsset {
 
     void createAnimators();
 
-    filament::Engine* mEngine;
-    utils::NameComponentManager* mNameManager;
-    utils::EntityManager* mEntityManager;
+    filament::Engine* const mEngine;
+    utils::NameComponentManager* const mNameManager;
+    utils::EntityManager* const mEntityManager;
+    NodeManager* const mNodeManager;
     std::vector<utils::Entity> mEntities; // sorted such that renderables come first
     std::vector<utils::Entity> mLightEntities;
     std::vector<utils::Entity> mCameraEntities;
```

Comment:
could these be FixedCapacityVector?

Reply:
I tried to do this in the past, I might try again on a rainy day but I would need to do the load in the two passes.  I do not know the counts apriori because the entities list is built recursively only from active root nodes.